app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/filtersControl/filtersControl.jsx

```diff
@@ -125,12 +133,24 @@ export class FiltersControl extends Component {
       page: 1,
       size: 10,
       searchValue: false,
+      activeFilter: null,
```

Comment:
Redundant state property. You already get all necessary info from props (`activeLaunchFilter` - for default filter, `value` - for current filter selected by user in wizard modal).